packages/web/config/ibc-assets.ts

```diff
@@ -141,6 +160,32 @@ export const IBCAssetInfos: (IBCAsset & {
       },
     ]
   : [
+      {
+        counterpartyChainId: "nomic-stakenet-3",
+        sourceChannelId: "channel-6897",
+        destChannelId: "channel-1",
+        coinMinimalDenom: "usat",
+        sourceChainNameOverride: "Bitcoin",
+        isVerified: true,
+        originBridgeInfo: {
+          bridge: "nomic" as const,
+          wallets: [],
+          method: "deposit-address" as const,
+          sourceChainTokens: [
+            {
+              id: "Bitcoin" as const,
+              logoUrl: "/networks/bitcoin.svg",
+            },
+          ],
+        },
+      },
+      {
+        counterpartyChainId: "mars-1",
+        sourceChannelId: "channel-557",
+        destChannelId: "channel-1",
+        coinMinimalDenom: "umars",
```

Comment:
I'm assuming duplicating $MARS here is a typo?

Reply:
Good catch! removed it on my latest commit